Firestore/Source/Public/FIRTimestampBehavior.h

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
I had to split out this enum. Otherwise, I ran into a situation where `FSTFieldValue.h` would have to import `FIRFirestoreSettings.h`, and XCode started complaining about everything inside `FIRFirestoreSettings.h` being redefined. I'm not sure how Objective C deals with double imports, perhaps my "fix" is wrong.

Reply:
Because it's now just a boolean flag, this is no longer an issue.